File: php/myproject/test/oop/student.php
<?php
class Students{
   public  $roll;
   public  $name;
   public  $subject;
    
   public function __construct($roll1,$name1,$subject1){
       $this->roll=$roll1;
       $this->name=$name1;
       $this->subject=$subject1;
       
   }
    
   public function getStudent(){
       echo "student details<br>";
       echo "<br>roll  ".$this->roll;
       echo "<br>name  ".$this->name;
       echo "<br>subject  ".$this->subject;
   }
}

$student=new Students(1001,'ram kumar','math');
$student->getStudent();
?>

File: corejava/project1/src/Student.java
public class Student {
private    int roll;
 private    String name;
    private   String subject;

    public static void main(String[] args) {
        Student student1=new Student();

        System.out.println("default value of student");
        System.out.println("roll is "+student1.roll);
        System.out.println("name is "+student1.name);
        System.out.println("subject is "+student1.subject);

        student1.roll=100001;
        student1.name="ram kumar";
        student1.subject="math";

        System.out.println("after initialisation value of student1 ");
        System.out.println("roll is "+student1.roll);
        System.out.println("name is "+student1.name);
        System.out.println("subject is "+student1.subject);
    }
}

